docker-compose.yml

```diff
@@ -625,6 +647,35 @@ services:
 
   # TODO: When adding E2E specs for Whitehall, it is likely we'll need to create a service for the Whitehall workers.
 
+  content-tagger: &content-tagger
+    build: apps/content-tagger
+    depends_on:
+      - content-tagger-worker
+      - diet-error-handler
+      - publishing-api
+    links:
+      - nginx-proxy:error-handler.dev.gov.uk
+      - nginx-proxy:publishing-api.dev.gov.uk
+      - postgres
+      - redis
+    environment:
+      << : *govuk-app
+      SENTRY_CURRENT_ENV: content-tagger
+      VIRTUAL_HOST: content-tagger.dev.gov.uk
+    ports:
+      - "3116"
+    volumes:
+      - ./apps/content-tagger/log:/app/log
+
+  content-tagger-worker:
+    << : *content-tagger
+    command: bundle exec sidekiq -C ./config/sidekiq.yml
```

Comment:
Do we need environment here? not sure if this will cause problems as it'll have a VIRTUAL_HOST env var and nginx-proxy will try do something with it.

Reply:
Good catch.  Been accessing it via the port so didn't run into the VIRTUAL_HOST problem.